src/util/versionstore.cpp

```diff
@@ -19,32 +19,50 @@
 #undef WIN32
 #endif
 
-#include <lame/lame.h>
 #include <FLAC/format.h>
 #include <chromaprint.h>
+#include <lame/lame.h>
+#include <portaudio.h>
 #include <rubberband/RubberBandStretcher.h>
-#include <taglib/taglib.h>
 #include <sndfile.h>
+#include <taglib/taglib.h>
 #include <vorbis/codec.h>
-#include <portaudio.h>
 
-#include "defs_version.h"
-#include "build.h" // Generated by SCons
+#define VERSION_STORE
+#include "version.h"
+
+namespace {
+
+const QString kMixxxVersion = QStringLiteral(MIXXX_VERSION);
+const QString kMixxx = QStringLiteral("Mixxx");
+const QString kGitBranch = QStringLiteral(GIT_BRANCH);
+#ifdef BUILD_REV
+const QString kBuildRev = QStringLiteral(BUILD_REV);
+#else
+const QString kBuildRev;
+#endif
+#ifdef BUILD_FLAGS
```

Comment:
Where does this come from? Is this ever defined? In the root of the repo, doesn't find the definition:
```
$ grep -Hrn BUILD_FLAGS *
src/util/versionstore.cpp:44:#ifdef BUILD_FLAGS
src/util/versionstore.cpp:45:const QString kBuildFlags = QStringLiteral(BUILD_FLAGS);
```

Reply:
This is a legacy feature from Scons, we have not yet ported to cmake. It was used when we have tweaked the optimization level. I can imagine that this is still useful to dig down performance issues with Disto builds like missing neon flag on arm builds.  

Repair or remove

Reply:
Repaired :-)